src/hotspot/share/gc/parallel/psPromotionManager.cpp

```diff
@@ -237,7 +237,9 @@ void PSPromotionManager::drain_stacks_depth(bool totally_drain) {
     // Drain overflow stack first, so other threads can steal from
     // claimed stack while we work.
```

Comment:
Pre-existing: not sure what a "claimed stack" is, maybe replace by "public stack" (or queue?). (The class is called "TaskQueue" after all).